src/app/core/interceptor/auth-interceptor.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import {
+  HttpInterceptor,
+  HttpRequest,
+  HttpHandler,
+  HttpEvent,
+  HttpHeaders,
+} from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class AuthInterceptorService implements HttpInterceptor {
+  constructor() {}
+
+  intercept(
+    req: HttpRequest<any>,
+    next: HttpHandler
+  ): Observable<HttpEvent<any>> {
+    let headers;
+    const jwtToken = localStorage.getItem('jwt-token');
```

Comment:
make constant variables for all the local storage key
like: LOCAL_STORAGE_JWT_TOKEN = 'jwt-token'